integrationTests/src/test/java/org/sonarqube/gradle/GradleTest.java

```diff
@@ -332,4 +330,43 @@ public void testKotlinJvmWithSubmoduleProject() throws Exception {
     assertThat(binaries).containsExactly(baseDir.resolve("submodule/build/classes/kotlin/main").toString());
     assertThat(sources).containsExactly(baseDir.resolve("submodule/src/main/kotlin").toString());
   }
+
+  @Test
+  public void testScanAllOnMultiModuleWithSubModulesProject() throws Exception {
```

Comment:
Can the title of the test be more explicit in what is expected to happen?
```suggestion
  public void testScanAllOnMultiModuleWithSubModulesProjectCollectsTheExpectedSources() throws Exception {
```